File: ALODWebUtility.CS/Workflow/SignStartedEventArgs.cs
using System;

namespace ALODWebUtility.Workflow
{
    public class SignStartedEventArgs : EventArgs
    {
        private bool _cancel = false;
        private string _comments;
        private byte _dbSign;
        private short _optionId;
        private int _refId;
        private int _secondaryId;
        private int _statusIn;
        private int _statusOut;

        public bool Cancel
        {
            get { return _cancel; }
            set { _cancel = value; }
        }

        public string Comments
        {
            get { return _comments; }
            set { _comments = value; }
        }

        public byte DBSign
        {
            get { return _dbSign; }
            set { _dbSign = value; }
        }

        public short OptionId
        {
            get { return _optionId; }
            set { _optionId = value; }
        }

        public int RefId
        {
            get { return _refId; }
            set { _refId = value; }
        }

        public int SecondaryId
        {
            get { return _secondaryId; }
            set { _secondaryId = value; }
        }

        public int StatusIn
        {
            get { return _statusIn; }
            set { _statusIn = value; }
        }

        public int StatusOut
        {
            get { return _statusOut; }
            set { _statusOut = value; }
        }
    }
}

File: ALODWebUtility/Workflow/SignStartedEventArgs.vb
Namespace Worklfow

    Public Class SignStartedEventArgs
        Inherits EventArgs

        Private _cancel As Boolean = False
        Private _comments As String
        Private _dbSign As Byte
        Private _optionId As Short
        Private _refId As Integer
        Private _secondaryId As Integer
        Private _statusIn As Integer
        Private _statusOut As Integer

        Public Property Cancel() As Boolean
            Get
                Return _cancel
            End Get
            Set(ByVal value As Boolean)
                _cancel = value
            End Set
        End Property

        Public Property Comments() As String
            Get
                Return _comments
            End Get
            Set(ByVal value As String)
                _comments = value
            End Set
        End Property

        Public Property DBSign() As Byte
            Get
                Return _dbSign
            End Get
            Set(ByVal value As Byte)
                _dbSign = value
            End Set
        End Property

        Public Property OptionId() As Short
            Get
                Return _optionId
            End Get
            Set(ByVal value As Short)
                _optionId = value
            End Set
        End Property

        Public Property RefId() As Integer
            Get
                Return _refId
            End Get
            Set(ByVal value As Integer)
                _refId = value
            End Set
        End Property

        Public Property SecondaryId() As Integer
            Get
                Return _secondaryId
            End Get
            Set(ByVal value As Integer)
                _secondaryId = value
            End Set
        End Property

        Public Property StatusIn() As Integer
            Get
                Return _statusIn
            End Get
            Set(ByVal value As Integer)
                _statusIn = value
            End Set
        End Property

        Public Property StatusOut() As Integer
            Get
                Return _statusOut
            End Get
            Set(ByVal value As Integer)
                _statusOut = value
            End Set
        End Property

    End Class

End Namespace